lms/static/js/spec/main.js

```diff
@@ -1,6 +1,7 @@
 (function(requirejs, define) {
     // TODO: how can we share the vast majority of this config that is in common with CMS?
     requirejs.config({
+        waitSeconds: 120,
```

Comment:
Why is waitSeconds being set to 2 minutes? I am very concerned because this will impact all tests (and could make them run very slowly). The default is 7 seconds.